tests/docker_wrapper/mod.rs

```diff
@@ -17,7 +17,7 @@ pub struct PostgresDocker {
 impl PostgresDocker {
     pub fn new() -> Result<PostgresDocker, Box<Error>> {
         let mut pg_docker = PostgresDocker { ip: "".to_string() };
-        try!(pg_docker.setup());
+        pg_docker.setup()?;
```

Comment:
for the tests, I recently used https://github.com/testcontainers/testcontainers-rs

not perfect, but not too bad to remove those wrappers

Reply:
thanks! i'll keep that in mind.